components/logins/android/src/main/java/mozilla/appservices/logins/DatabaseLoginsStorage.kt

```diff
@@ -111,60 +111,40 @@ class DatabaseLoginsStorage(private val dbPath: String) : AutoCloseable {
     }
 
     @Throws(LoginsStorageException::class)
-    fun list(): List<Login> {
+    fun decryptAndFixupLogin(encKey: String, login: Login): LoginPayload {
```

Comment:
Maybe we can commit to this just being `decrypt()` since Sammy is going to rewrite the migration to validate it?  I'm not sure on that at all, just wanted to raise the question.

Reply:
We should postpone this decision until after the current effort so shipping the migration isn't delayed and any fixup-related regressions are isolated. But I hope that we are able to eventually remove the fixup logic here.